src/Traits/DetectsChanges.php

```diff
@@ -154,4 +157,13 @@ protected static function getRelatedModelAttributeValue(Model $model, string $at
 
         return ["{$relatedModelName}.{$relatedAttribute}" => $relatedModel->$relatedAttribute ?? null];
     }
+
+    protected static function jsonLogAttribute(Model $model, string $attribute)
```

Comment:
I would like to name the method like `getRelatedModelAttributeValue()`. My idea would be `getModelAttributeJsonValue()`.